docs/UsersGuide/source/Reference/FAQ.rst

```diff
@@ -0,0 +1,270 @@
+
+.. _ChangePhysics:
+
+=========================================================
+How do I change the Physics Suite Definition File (SDF)?
+=========================================================
+
+The SDF is set in the ``workflow:`` section of the ``config.yaml`` file using the variable ``CCPP_PHYS_SUITE``. The four supported physics suites for the SRW Application as of the v2.1.0 release are:
+
+.. code-block:: console
+   
+   FV3_GFS_v16
+   FV3_RRFS_v1beta
+   FV3_HRRR
+   FV3_WoFS_v0
+
+When users run the ``generate_FV3LAM_wflow.py`` script, the SDF file is copied from its location in the forecast
+model directory to the experiment directory ``$EXPTDIR``. For more information on the :term:`CCPP` physics suite parameters, see :numref:`Section %s <CCPP_Params>`.
+
+.. _ChangeGrid:
+
+===========================
+How do I change the grid?
+===========================
+
+To change the predefined grid, modify the ``PREDEF_GRID_NAME`` variable in the ``task_run_fcst:`` section of the ``config.yaml`` script (see :numref:`Section %s <UserSpecificConfig>` for details on creating and modifying the ``config.yaml`` file). The four supported predefined grids as of the SRW Application v2.1.0 release are:
+
+.. code-block:: console
+   
+   RRFS_CONUS_3km
+   RRFS_CONUS_13km
+   RRFS_CONUS_25km
+   SUBCONUS_Ind_3km
+
+However, users can choose from a variety of predefined grids listed in :numref:`Section %s <PredefGrid>`. An option also exists to create a user-defined grid, with information available in :numref:`Section %s <UserDefinedGrid>`. However, the user-defined grid option is not fully supported as of the v2.1.0 release and is provided for informational purposes only. 
+
+.. _SetTasks:
+
+===============================================
+How can I select which workflow tasks to run? 
+===============================================
+
+The ``/parm/wflow`` directory contains several ``YAML`` files that configure different workflow task groups. Each task group file contains a number of tasks that are typically run together. :numref:`Table %s <task-group-files>` describes each of the task groups. 
+
+.. _task-group-files:
+
+.. list-table:: Task group files
+   :widths: 20 50
+   :header-rows: 1
+
+   * - File
+     - Function
+   * - aqm_post.yaml
+     - SRW-AQM post-processing tasks
+   * - aqm_prep.yaml
+     - SRW-AQM pre-processing tasks
+   * - coldstart.yaml
+     - Tasks required to run a cold-start forecast
+   * - da_data_preproc.yaml
+     - Preprocessing tasks for RRFS `DA <data assimilation>`.
+   * - plot.yaml
+     - Plotting tasks
+   * - post.yaml
+     - Post-processing tasks
+   * - prdgen.yaml
+     - 
+   * - prep.yaml
+     - Pre-processing tasks
+   * - verify_det.yaml
+     - Deterministic verification tasks
+   * - verify_ens.yaml
+     - Ensemble verification tasks
+   * - verify_pre.yaml
+     - Verification pre-processing tasks
+
+.. COMMENT: What does prdgen.yaml do? 
```

Comment:
It creates small domain products from big domain created by UPP. Used for various applications, and covers CONUS, Alaska, HI, PR...